src/LiveSplit.Text/UI/Components/TextComponentSettings.Designer.cs

```diff
@@ -548,14 +551,27 @@ private void InitializeComponent()
             this.chkTwoRows.Text = "Display 2 Rows";
             this.chkTwoRows.UseVisualStyleBackColor = true;
             // 
+            // chkCustomVariable
+            // 
+            this.chkCustomVariable.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Left | System.Windows.Forms.AnchorStyles.Right)));
+            this.chkCustomVariable.AutoSize = true;
+            this.chkCustomVariable.Location = new System.Drawing.Point(7, 35);
+            this.chkCustomVariable.Margin = new System.Windows.Forms.Padding(7, 3, 3, 3);
+            this.chkCustomVariable.Name = "chkCustomVariable";
+            this.chkCustomVariable.Size = new System.Drawing.Size(149, 17);
+            this.chkCustomVariable.TabIndex = 3;
+            this.chkCustomVariable.Text = "Custom Variable";
```

Comment:
Is it possible to add a tooltip to explain this setting?
It's not clear that this is only for the Auto Splitting Runtime for autosplitters that support custom variables, and also that you need to put the variable name where you'd normally put the text

Reply:
Either that or make the labels clearer ("Autosplitter Custom Variable" for the checkbox, and rename the Text2 label to "Custom Variable Name" if the checkbox is checked)?

Reply:
Okay, I've added a ToolTip and made it rename the Text2 label if the checkbox is checked.